libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.h

```diff
@@ -54,7 +53,7 @@ extern NSString * const kUserAgentCredentialsDictKey;
  Callback block definition for OAuth plugin auth success.
  */
 typedef void (^SFOAuthPluginAuthSuccessBlock)(SFOAuthInfo *_Nullable, NSDictionary *);
-
+typedef void (^SFOAuthPluginFailureBlock)(SFOAuthInfo *, NSError *);
```

Comment:
Added this definition here for consistency. Essentially these blocks are the same as defined by SFUserAccountManager  login api requirements